spec/models/validations/sales/setup_validations_spec.rb

```diff
@@ -7,15 +7,6 @@
 
   describe "#validate_saledate_collection_year" do
     context "with sales_in_crossover_period == false" do
-      before do
```

Comment:
Do we need to force this to be true some other way?

Reply:
yup